convert2rhel/subscription.py

```diff
@@ -286,6 +288,22 @@ def args(self):
 
         return args
 
+    @property
+    def connection_opts(self):
+        connection_opts = {}
+
+        if self.server_url:
+            url_parts = urllib.parse.urlsplit(self.server_url, allow_fragments=False)
+            connection_opts = {"host": url_parts.hostname}
+            connection_opts = {"port": url_parts.port}
```

Comment:
if no port was specified, there is no `port` in `url_part`

Reply:
Fixed